01-JS-DOM-APIs/assets/js/basic.js

```diff
@@ -0,0 +1,151 @@
+window.onload = function () {
+  var sectionElement = document.getElementById('helloWorld');
+
+  if(sectionElement.style.display == "block"){
+    sectionElement.style.display = "none";
+  }else {
+    var timer = setInterval(function(){
+      sectionElement.style.display = "block";
+      sectionElement.className = "faded";
+    }, 1000);
+  }
+}
+
+  var button = document.getElementById('button');
+  var response = document.getElementById('response');
+  var gitBtn = document.getElementById('gitButton');
+
+
+  var config = {
+    method: "GET",
+    url: "http://api.icndb.com/jokes/random",
+    async_value: true
```

Comment:
I'd name it as `async` but if you want to keep this variable name I'd call it like `asyncValue` as a good practice